src/web/forms/DynamicForm.js

```diff
@@ -63,10 +63,16 @@ function makeCreateChildFields(
 
     const elements = !styles ? [] : styles.map((field, i) => {
       // Handle "show" prop
-      if (!checkVisibility(state, rootModel, field.show)) {
+      if (!referAndGetBool(state, rootModel, field.show)) {
         return null;
       }
 
+      // Translate "disabled" and "enabled" prop to boolean value
```

Comment:
Maybe it is more appropriate to name something like checkEnabledToBool? (as it seems its main purpose is for check disable/enable -> bool output)